memgpt/agent_store/db.py

```diff
@@ -156,6 +156,7 @@ class MessageModel(Base):
             # openai info
             role = Column(String, nullable=False)
             text = Column(String)  # optional: can be null if function call
```

Comment:
Here we should probably delete
`text = Column(String)  # optional: can be null if function call`

and replace with
`content = Column(JSON) # optional: multi-modal input`

which in the pydantic model is `Optional[Union[str, List[MultiModalMessagePart]]]`

but in the database itself is stored as an optional JSON field